publify_core/app/views/archives_sidebar/_content.html.erb

```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
```

Comment:
You did not check out master before you created this new branch. That is why your issue #1 changes are still on this branch. 

This is an unnecessary change, because this code has nothing to do with issue #2.